data/json/recipes/recipe_others.json

```diff
@@ -702,6 +702,23 @@
       [ [ "cordage_short", 2, "LIST" ], [ "filament", 40, "LIST" ] ]
     ]
   },
+  {
+    "type": "recipe",
+    "result": "makeshift_shovel",
+    "category": "CC_OTHER",
+    "subcategory": "CSC_OTHER_TOOLS",
+    "skill_used": "survival",
+    "skills_required": [ "fabrication", 1 ],
+    "difficulty": 2,
+    "time": "60 m",
+    "autolearn": true,
+    "qualities": [ { "id": "HAMMER", "level": 1 } ],
```

Comment:
- this also looks more like fabrication than survival
- I'm not sure how a bit of cordage is enough to make a shovel that holds up to any work.

I'd suggest adding drilling to the required tool qualities, and two spikes or six nails to the components, implying that you are putting holes in and riveting the thing onto the handle. Also I'd bump the hammer requirement to 2 (really for all the makeshift banged metal items you're listing. They'd by pretty hard to make by whacking something with a rock)

Reply:
A "makeshift pot", out of metal sheet, is a metal sheet with enough of a bend for water to be contained... right? Or am I missing something?

Reply:
yeah it seems to be a piece of sheet metal you've banged into a basin sort of shape. Sheet metal isn't that easy to bend in general, I don't think hammering 1 is realistically enough to do it.

Reply:
Updated it, but decided to use wire instead of spikes (they seem way too bulky for this purpose).